File: JavaProject/src/com/viva1/Variables.java
package com.viva1;

public class Variables {
	
		
	    static int sharedVar = 10;//common for all
	    int instanceVar = 20;//return above method and called through class object
	    private int instance = 30;

	    void display() {
	        int localVar = 30;
	        sharedVar=1;
	        System.out.println(localVar);
	    }
	    public int getInsatance() {//for private
	    	return instance;
	    }
	

}

File: PythonProject/variables/variables.py
class Variables:
    shared_var = 10  # Class variable (shared across all instances)

    def __init__(self):
        self.instance_var = 20  # Instance variable
        self.__instance = 30    # Private instance variable

    def display(self):
        local_var = 30
        Variables.shared_var = 1
        print(local_var)

    def get_instance(self):
        return self.__instance


# Equivalent of the Java 'main' method
if __name__ == "__main__":
    v = Variables()
    v.display()
    print(Variables.shared_var)
    print(v.instance_var)
    print(v.get_instance())
